### PR TITLE
Add conversion for boolean attribute

### PR DESCRIPTION
**Pull request by schweitzpgi**

### CODE REVIEW

File: lib/Target/LLVMIR/ModuleTranslation.cpp [hunk 1]
```diff
@@ -84,6 +84,8 @@ llvm::Constant *ModuleTranslation::getLLVMConstant(llvm::Type *llvmType,
     return llvm::ConstantFP::get(llvmType, floatAttr.getValue());
   if (auto funcAttr = attr.dyn_cast<SymbolRefAttr>())
     return functionMapping.lookup(funcAttr.getValue());
+  if (auto boolAttr = attr.dyn_cast<BoolAttr>())
```
Comment (River707) [inline lib/Target/LLVMIR/ModuleTranslation.cpp]: Can you add a test case to exercise this? (to test/Target/llvmir.mlir)

Comment (schweitzpgi) [reply lib/Target/LLVMIR/ModuleTranslation.cpp]: It appears that the problem we had may have been resolved another way.  We're still checking.

**Comment by antiagainst** [comment]
Hey @schweitzpgi, are you still working on this PR? You will need to sign the CLA before we can accept the contribution. :)

**Comment by schweitzpgi** [comment]
I think this was fixed a different way. It no longer seems to cause the same issue. Closing.

**Pull request closed without merging**